content/docs/studio/user-guide/projects-and-experiments/live-metrics-and-plots.md

```diff
@@ -0,0 +1,111 @@
+`Studio access token` section, click on `Generate new token`.
+
+### Regenerate access token
+
+You can regenerate the access token at any point. The old access token will no
+longer be authorized to send live metrics and plots to Iterative Studio.
+
+### Reset (delete) access token
+
+You can delete the access token when you no longer need it.
+
+The option to delete the access token is also available when you try to change
+your password. That is, you can reset all your access credentials (your password
+and the access token) at once. This is handy if you suspect that your account
+security may have been compromised.
+
+## Configure your model training CI job
+
+You should define the following environment variables in your CI job:
+
+- `STUDIO_REPO_URL`: URL for the Git repository connected to the project. E.g.,
+  `github:iterative/example-get-started`
+- `STUDIO_TOKEN`: Your access token. The access token must be present in any
+  request that sends data to the Iterative Studio ingestion endpoint. Requests
+  with missing or incorrect access tokens are rejected with an appropriate HTTP
+  error code and error message.
+
+One way to provide the above values to your CI job is to set them as GitHub
+secrets.
+
+Here is an example GitHub action.
+
+TODO: provide link to an example snippet (such as
+[this](https://github.com/iterative/test-dvclive-studio/blob/086a51d76c7983f24c091e1b007820916aa75e7d/.github/workflows/test_live_metrics.yaml#L17-L19))
+
+## Send the updates
+
+In the training CI job (which has been configured as detailed above), whenever
+you log your metrics or plots using [DVCLive], they will be automatically sent
+to Iterative Studio.
+
+Iterative Studio stores the live metrics and plots data in its database.
+
+## View live metrics and plots
+
+In the project table, the live metrics are displayed next to the Git commit
+corresponding to the experiment. Updates to the live metrics are highlighted in
+orange. The running experiments themselves are displayed with a `Running`
+indicator.
+
+TODO: Add a screenshot of the `Running` indicator
+
+The number of experiments with new updates to the live metrics values are
+displayed in the `Live` icon, which can also be used to filter and show only
+running experiments in the table.
+
+Live plots are displayed in the
+[plots pane](/doc/studio/user-guide/projects-and-experiments/visualize-and-compare#how-to-generate-plots).
+
+An experiment can have one of the following statuses:
+
+- **Running** - Iterative Studio expects to receive live metrics and plots for
+  these experiments.
+- **Completed** - Iterative Studio does expect to receive any more updates for
+  these experiments. Once the experiment concludes, live metrics or plots will
+  no longer be displayed for the experiment. You are expected to persist the
+  final results in Git. TODO: confirm if this statement is correct.
+
+- **Stopped/Error** - These experiments stopped due to some error.
```

Comment:
Can the error message/log be found somewhere?

Reply:
I don't think we have details of the errors. @mvshmakov is there any details we can provide here?

Reply:
I believe the release will not include the concept of stopped/errored experiment, only completed

Reply:
Oh ok. In that case, I should remove this from the docs. 
@daavoo what happens in case of an error - does it show up in Studio as running or completed?
cc @mvshmakov

Reply:
As @daavoo said already, we don't have a notion of an errored-out live experiment. If, say, the process on the remote machine shuts down, DVCLive just stops sending any events and we'll have a never-ending progressing live experiment.

Users are able to delete such experiments as it is usually obvious to them that the process failed. We've discussed errors on the previous live metrics sync and it was stated that it is unrealistic to expect DVCLive to handle all types of errors that may happen.

Reply:
ok, thanks @mvshmakov. I'll update the docs accordingly.

Reply:
I've updated it [here](https://github.com/iterative/dvc.org/pull/4012/commits/ea378ba5f6317a16b2d6f76a397774dcfa9336bb).

Reply:
Thanks guys! Resolving, I guess.